code/modules/clothing/shoes/boots.dm

```diff
@@ -3,6 +3,7 @@
 	desc = "High speed, low drag combat boots."
 	icon_state = "jackboots"
 	inhand_icon_state = "jackboots"
+	body_parts_covered = FEET|LEGS
```

Comment:
I figure normal jackboots should also cover legs, even though they have no armor, for consistency